CDC.WebNotes.Application/Services/AttachmentService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using AutoMapper;
+using CDC.WebNotes.Application.Contracts;
+using CDC.WebNotes.Data.Contracts;
+using CDC.WebNotes.Data.Repositories;
+using CDC.WebNotes.Domain.Notes;
+using CDC.WebNotes.Dto;
+using CDC.WebNotes.Dto.Notes;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace CDC.WebNotes.Application.Services
+{
+    public class AttachmentService : IAttachmentService
+    {
+        private readonly IAttachmentsRepository _attachmentRepository;
+        private readonly IMapper _mapper;
+
+        public AttachmentService(IAttachmentsRepository attachmentRepository,
+                                         IMapper mapper)
+        {
+            _mapper = mapper;
+            _attachmentRepository = attachmentRepository;
+        }
+
+        public async Task<AttachmentsResponseDto> GetAllAttachments()
```

Comment:
Shoud we get all attachments for Note by by noteId. pls update logig for this method